net/net.h

```diff
@@ -403,6 +403,9 @@ struct m0_net_domain {
          */
 	m0_bcount_t         nd_get_max_buffer_desc_size;
 
+	/** The network domain name which is to be initialiazed */
+	const char         *nd_name;
+
```

Comment:
I don't think we need this.
We can use xprt->nx_name

Reply:
The xprt->name  would be lnet/sock/libfabric which does not provide any info about the interface that we want to use.
Hence this was added. But after discussion with Nikita yesterday, there is some design change required for this. Hence will cancel this PR as of now